tests/test_client/tests.py

```diff
@@ -85,6 +87,52 @@ def test_post(self):
         self.assertEqual(response.templates[0].name, 'POST Template')
         self.assertContains(response, 'Data received')
 
+    def assert_json_response(self, response, method):
```

Comment:
We use CamelCase for assert methods to follow `unittest` convention, e.g. `assertJSONResponse`. Maybe it will be better to pass `method` and `data` instead of `response`, e.g.:
```python
def assertJSONResponse(method, data, method_name):
    response = method('/json_view/', data, content_type=JSON_CONTENT)
    self.assertEqual(response.status_code, 200)
    self.assertEqual(response.context['data'], 37)
    self.assertEqual(response.templates[0].name, '{} Template'.format(method))
    self.assertContains(response, 'Viewing {} page.'.format(method))

def test_json_post(self):
    assertJSONResponse(self.client.post, {'value': 37}, 'POST')

def test_json_put(self):
    assertJSONResponse(self.client.put, {'value': 37}, 'PUT')
...
```
Please also try to avoid temporary variables like `post_data`, `put_data`, etc.

Reply:
Done